vitest.config.ts

```diff
@@ -1,19 +1,8 @@
-import path from 'path'
-import { defineConfig } from 'vitest/config'
-import Vue from '@vitejs/plugin-vue'
+import { defineVitestConfig } from '@nuxt/test-utils/config'
 
-export default defineConfig({
-    plugins: [Vue()],
+export default defineVitestConfig({
```

Comment:
I don't like how this completely replaces the whole native vitest config with their own nuxt version :( 
Is there any way to use nuxt within the original config style?

Reply:
@ermish The answer seems to be no.
Based on this link and article https://masteringnuxt.com/blog/unit-testing-in-nuxt

And the official docs which include this:

![image](https://github.com/user-attachments/assets/d97cab9a-ff16-432b-8e2e-84d646968fee)